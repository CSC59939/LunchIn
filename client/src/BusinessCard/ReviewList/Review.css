.review-card{
  display: flex;
  width: 100%;
  height: 100%;
}
.review-card-media{
  object-fit: cover;
  width: 100%;
  height: 100%;
}
.review-content{
  padding: 1%;
  overflow: auto;
}
