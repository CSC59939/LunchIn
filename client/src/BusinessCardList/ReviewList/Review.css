.review-card{

}
.review-card-media{
  width: 200px;
  height: 150px;
}
