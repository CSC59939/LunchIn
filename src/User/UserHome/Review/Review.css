.review-card{
  max-width: 345px,
}
.review-card-media{
  width: 200px;
  height: 150px;
}
