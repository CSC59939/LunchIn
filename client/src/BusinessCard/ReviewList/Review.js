import React, { Component } from "react";
import Card from '@material-ui/core/Card';
import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';
import CircularProgress from '@material-ui/core/CircularProgress';
import "./Review.css";

class Review extends Component {
  render() {
    if(this.props.review){
      const r = this.props.review;
      return (
        <Card className="review-card">
            <Grid container>
              <Grid item xs={4} sm={4} md={4} lg={4} xl={4}>
                <img
                  className="review-card-media"
                  src={r.user.image_url}
                  alt="Failed to load review URL."
                />
              </Grid>
              <Grid item xs={8} sm={8} md={8} lg={8} xl={8}
                className="review-content"
                >
                <Typography variant="h6" component="h6">
                  {r.user.name}
                </Typography>
                <Typography component="p">
                  {r.time_created.split(' ')[0]} Rating: {r.rating} <br />
                  {r.text}
                </Typography>
              </Grid>
            </Grid>
        </Card>
      );
    }
    else{
      return (<CircularProgress />);
    }
  }
}
export default Review;
