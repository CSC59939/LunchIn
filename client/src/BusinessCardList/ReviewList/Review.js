import React, { Component } from "react";
import Card from '@material-ui/core/Card';
import CardActionArea from '@material-ui/core/CardActionArea';
import CardContent from '@material-ui/core/CardContent';
import CardMedia from '@material-ui/core/CardMedia';
import Typography from '@material-ui/core/Typography';
import CircularProgress from '@material-ui/core/CircularProgress';
import "./Review.css";

class Review extends Component {
  render() {
    if(this.props.review){
      return (
        <Card className="review-card">
            <CardMedia
              className="review-card-media"
              image={this.props.review.user.image_url}
              title={this.props.review.user.image_url}
            />
            <Typography component="p">
              {this.props.review.user.name} <br />
              {this.props.review.time_created} <br />
              {this.props.review.rating} <br />
              {this.props.review.text} <br />
            </Typography>
        </Card>
      );
    }
    else{
      return (<CircularProgress />);
    }
  }
}
export default Review;
