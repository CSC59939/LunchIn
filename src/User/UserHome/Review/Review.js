import React, { Component } from "react";
import "./Review.css";

/*
  This component requires two properties: term, location
*/
class Review extends Component{
  constructor(props){
    super(props);
    this.state = {

    };
    this.renderReview = this.renderReview.bind(this);
    this.getReviews = this.getReviews.bind(this);
    this.getReviews();
  }

  getReviews(){
		fetch(`/api/yelp?term=${this.props.term}&location=${this.props.location}`)
			.then(response => response.json())
			.then(data => {
				console.log(data);
				this.setState({businesses: data.jsonBody.businesses});
			})
			.catch(e => console.log(e));
  }

  renderReview(review){
    return (
      <div className="review" key={review.id}>
        {/*console.log(review)*/}
        <div className="review-name">{review.name}</div>
        <div className="review-price">Price: {review.price}</div>
        <div className="review-rating">Rating: {review.rating}</div>
        <div className="review-review_count">Count: {review.review_count}</div>
        <div><img src={review.image_url} className="review-img"></img></div>
      </div>
    );
  }

  render(){
    return (
      <div className="review-list">
        { this.props.term
          ? (
              <div className="selected-food">
                We have selected the following food: {this.props.term}
              </div>
            )
          : (
              <div className="error">"Unable to pick a lunch..."</div>
            )
        }
        { this.state.businesses
          ? (this.state.businesses.map(this.renderReview))
          : ("")
        }
      </div>
    );
  }
}
export default Review;
