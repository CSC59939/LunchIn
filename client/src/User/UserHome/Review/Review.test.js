import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Review from "./Review";

describe("Review Component", () => {
  describe("When provided with no info", () => {
    it("Should render correctly", () => {
      const reviewInstance = shallow(
        <Review />
      );
      expect(reviewInstance).toMatchSnapshot();
    });
  });
  describe("When provided with review info", () => {
    it("Should render correctly", () => {
      const businesses = [
        {
          "rating": 4.5,
          "price": "$$",
          "phone": "+14154212337",
          "id": "molinari-delicatessen-san-francisco",
          "categories": [
            {
              "alias": "delis",
              "title": "Delis"
            }
          ],
          "review_count": 910,
          "name": "Molinari Delicatessen",
          "url": "https://www.yelp.com/biz/molinari-delicatessen-san-francisco",
          "coordinates": {
            "latitude": 37.7983818054199,
            "longitude": -122.407821655273
          },
          "image_url": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
          "location": {
            "city": "San Francisco",
            "country": "US",
            "address2": "",
            "address3": "",
            "state": "CA",
            "address1": "373 Columbus Ave",
            "zip_code": "94133"
          }
        },
        {
          "rating": 4,
          "image_url": "",
          "name": "North India Restaurant",
          "url": "https://www.yelp.com/biz/north-india-restaurant-san-francisco",
          "review_count": 551,
          "coordinates": {
            "latitude": null,
            "longitude": null
          },
          "id": "north-india-restaurant-san-francisco",
          "categories": [
            {
              "alias": "indpak",
              "title": "Indian"
            }
          ]
        }
      ];
      const reviewInstance = shallow(
        <Review businesses={businesses} />
      );
      expect(reviewInstance).toMatchSnapshot();
    });
  });
});
