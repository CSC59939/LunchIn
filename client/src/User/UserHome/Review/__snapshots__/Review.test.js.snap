// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Component When provided with no info Should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Review />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": null,
      "className": "review-list",
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": null,
        "className": "review-list",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`Review Component When provided with review info Should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Review
    businesses={
      Array [
        Object {
          "categories": Array [
            Object {
              "alias": "delis",
              "title": "Delis",
            },
          ],
          "coordinates": Object {
            "latitude": 37.7983818054199,
            "longitude": -122.407821655273,
          },
          "id": "molinari-delicatessen-san-francisco",
          "image_url": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
          "location": Object {
            "address1": "373 Columbus Ave",
            "address2": "",
            "address3": "",
            "city": "San Francisco",
            "country": "US",
            "state": "CA",
            "zip_code": "94133",
          },
          "name": "Molinari Delicatessen",
          "phone": "+14154212337",
          "price": "$$",
          "rating": 4.5,
          "review_count": 910,
          "url": "https://www.yelp.com/biz/molinari-delicatessen-san-francisco",
        },
        Object {
          "categories": Array [
            Object {
              "alias": "indpak",
              "title": "Indian",
            },
          ],
          "coordinates": Object {
            "latitude": null,
            "longitude": null,
          },
          "id": "north-india-restaurant-san-francisco",
          "image_url": "",
          "name": "North India Restaurant",
          "rating": 4,
          "review_count": 551,
          "url": "https://www.yelp.com/biz/north-india-restaurant-san-francisco",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <WithStyles(Card)
          className="review-card"
        >
          <WithStyles(CardActionArea)>
            <WithStyles(CardMedia)
              className="review-card-media"
              image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
              title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
            />
            <WithStyles(CardContent)>
              <WithStyles(Typography)
                component="h2"
                gutterBottom={true}
                variant="h5"
              >
                Molinari Delicatessen
              </WithStyles(Typography)>
              <WithStyles(Typography)
                component="p"
              >
                Price: 
                $$
                 
                <br />
                Rating: 
                4.5
                 
                <br />
                Review Counts: 
                910
              </WithStyles(Typography)>
            </WithStyles(CardContent)>
          </WithStyles(CardActionArea)>
        </WithStyles(Card)>,
        <WithStyles(Card)
          className="review-card"
        >
          <WithStyles(CardActionArea)>
            <WithStyles(CardMedia)
              className="review-card-media"
              image=""
              title=""
            />
            <WithStyles(CardContent)>
              <WithStyles(Typography)
                component="h2"
                gutterBottom={true}
                variant="h5"
              >
                North India Restaurant
              </WithStyles(Typography)>
              <WithStyles(Typography)
                component="p"
              >
                Price: 
                 
                <br />
                Rating: 
                4
                 
                <br />
                Review Counts: 
                551
              </WithStyles(Typography)>
            </WithStyles(CardContent)>
          </WithStyles(CardActionArea)>
        </WithStyles(Card)>,
      ],
      "className": "review-list",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "Molinari Delicatessen",
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(CardActionArea)>
            <WithStyles(CardMedia)
              className="review-card-media"
              image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
              title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
            />
            <WithStyles(CardContent)>
              <WithStyles(Typography)
                component="h2"
                gutterBottom={true}
                variant="h5"
              >
                Molinari Delicatessen
              </WithStyles(Typography)>
              <WithStyles(Typography)
                component="p"
              >
                Price: 
                $$
                 
                <br />
                Rating: 
                4.5
                 
                <br />
                Review Counts: 
                910
              </WithStyles(Typography)>
            </WithStyles(CardContent)>
          </WithStyles(CardActionArea)>,
          "className": "review-card",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <WithStyles(CardMedia)
                className="review-card-media"
                image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
                title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
              />,
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  Molinari Delicatessen
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                  $$
                   
                  <br />
                  Rating: 
                  4.5
                   
                  <br />
                  Review Counts: 
                  910
                </WithStyles(Typography)>
              </WithStyles(CardContent)>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "review-card-media",
                "image": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
                "title": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <WithStyles(Typography)
                    component="h2"
                    gutterBottom={true}
                    variant="h5"
                  >
                    Molinari Delicatessen
                  </WithStyles(Typography)>,
                  <WithStyles(Typography)
                    component="p"
                  >
                    Price: 
                    $$
                     
                    <br />
                    Rating: 
                    4.5
                     
                    <br />
                    Review Counts: 
                    910
                  </WithStyles(Typography)>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Molinari Delicatessen",
                    "component": "h2",
                    "gutterBottom": true,
                    "variant": "h5",
                  },
                  "ref": null,
                  "rendered": "Molinari Delicatessen",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      "Price: ",
                      "$$",
                      " ",
                      <br />,
                      "Rating: ",
                      4.5,
                      " ",
                      <br />,
                      "Review Counts: ",
                      910,
                    ],
                    "component": "p",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Price: ",
                    "$$",
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "Rating: ",
                    4.5,
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "Review Counts: ",
                    910,
                  ],
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "North India Restaurant",
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(CardActionArea)>
            <WithStyles(CardMedia)
              className="review-card-media"
              image=""
              title=""
            />
            <WithStyles(CardContent)>
              <WithStyles(Typography)
                component="h2"
                gutterBottom={true}
                variant="h5"
              >
                North India Restaurant
              </WithStyles(Typography)>
              <WithStyles(Typography)
                component="p"
              >
                Price: 
                 
                <br />
                Rating: 
                4
                 
                <br />
                Review Counts: 
                551
              </WithStyles(Typography)>
            </WithStyles(CardContent)>
          </WithStyles(CardActionArea)>,
          "className": "review-card",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <WithStyles(CardMedia)
                className="review-card-media"
                image=""
                title=""
              />,
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  North India Restaurant
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                   
                  <br />
                  Rating: 
                  4
                   
                  <br />
                  Review Counts: 
                  551
                </WithStyles(Typography)>
              </WithStyles(CardContent)>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "review-card-media",
                "image": "",
                "title": "",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <WithStyles(Typography)
                    component="h2"
                    gutterBottom={true}
                    variant="h5"
                  >
                    North India Restaurant
                  </WithStyles(Typography)>,
                  <WithStyles(Typography)
                    component="p"
                  >
                    Price: 
                     
                    <br />
                    Rating: 
                    4
                     
                    <br />
                    Review Counts: 
                    551
                  </WithStyles(Typography)>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "North India Restaurant",
                    "component": "h2",
                    "gutterBottom": true,
                    "variant": "h5",
                  },
                  "ref": null,
                  "rendered": "North India Restaurant",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      "Price: ",
                      undefined,
                      " ",
                      <br />,
                      "Rating: ",
                      4,
                      " ",
                      <br />,
                      "Review Counts: ",
                      551,
                    ],
                    "component": "p",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Price: ",
                    undefined,
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "Rating: ",
                    4,
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "Review Counts: ",
                    551,
                  ],
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <WithStyles(Card)
            className="review-card"
          >
            <WithStyles(CardActionArea)>
              <WithStyles(CardMedia)
                className="review-card-media"
                image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
                title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
              />
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  Molinari Delicatessen
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                  $$
                   
                  <br />
                  Rating: 
                  4.5
                   
                  <br />
                  Review Counts: 
                  910
                </WithStyles(Typography)>
              </WithStyles(CardContent)>
            </WithStyles(CardActionArea)>
          </WithStyles(Card)>,
          <WithStyles(Card)
            className="review-card"
          >
            <WithStyles(CardActionArea)>
              <WithStyles(CardMedia)
                className="review-card-media"
                image=""
                title=""
              />
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  North India Restaurant
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                   
                  <br />
                  Rating: 
                  4
                   
                  <br />
                  Review Counts: 
                  551
                </WithStyles(Typography)>
              </WithStyles(CardContent)>
            </WithStyles(CardActionArea)>
          </WithStyles(Card)>,
        ],
        "className": "review-list",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "Molinari Delicatessen",
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(CardActionArea)>
              <WithStyles(CardMedia)
                className="review-card-media"
                image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
                title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
              />
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  Molinari Delicatessen
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                  $$
                   
                  <br />
                  Rating: 
                  4.5
                   
                  <br />
                  Review Counts: 
                  910
                </WithStyles(Typography)>
              </WithStyles(CardContent)>
            </WithStyles(CardActionArea)>,
            "className": "review-card",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <WithStyles(CardMedia)
                  className="review-card-media"
                  image="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
                  title="http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg"
                />,
                <WithStyles(CardContent)>
                  <WithStyles(Typography)
                    component="h2"
                    gutterBottom={true}
                    variant="h5"
                  >
                    Molinari Delicatessen
                  </WithStyles(Typography)>
                  <WithStyles(Typography)
                    component="p"
                  >
                    Price: 
                    $$
                     
                    <br />
                    Rating: 
                    4.5
                     
                    <br />
                    Review Counts: 
                    910
                  </WithStyles(Typography)>
                </WithStyles(CardContent)>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "className": "review-card-media",
                  "image": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
                  "title": "http://s3-media4.fl.yelpcdn.com/bphoto/6He-NlZrAv2mDV-yg6jW3g/o.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(Typography)
                      component="h2"
                      gutterBottom={true}
                      variant="h5"
                    >
                      Molinari Delicatessen
                    </WithStyles(Typography)>,
                    <WithStyles(Typography)
                      component="p"
                    >
                      Price: 
                      $$
                       
                      <br />
                      Rating: 
                      4.5
                       
                      <br />
                      Review Counts: 
                      910
                    </WithStyles(Typography)>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Molinari Delicatessen",
                      "component": "h2",
                      "gutterBottom": true,
                      "variant": "h5",
                    },
                    "ref": null,
                    "rendered": "Molinari Delicatessen",
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": Array [
                        "Price: ",
                        "$$",
                        " ",
                        <br />,
                        "Rating: ",
                        4.5,
                        " ",
                        <br />,
                        "Review Counts: ",
                        910,
                      ],
                      "component": "p",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Price: ",
                      "$$",
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Rating: ",
                      4.5,
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Review Counts: ",
                      910,
                    ],
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "North India Restaurant",
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(CardActionArea)>
              <WithStyles(CardMedia)
                className="review-card-media"
                image=""
                title=""
              />
              <WithStyles(CardContent)>
                <WithStyles(Typography)
                  component="h2"
                  gutterBottom={true}
                  variant="h5"
                >
                  North India Restaurant
                </WithStyles(Typography)>
                <WithStyles(Typography)
                  component="p"
                >
                  Price: 
                   
                  <br />
                  Rating: 
                  4
                   
                  <br />
                  Review Counts: 
                  551
                </WithStyles(Typography)>
              </WithStyles(CardContent)>
            </WithStyles(CardActionArea)>,
            "className": "review-card",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <WithStyles(CardMedia)
                  className="review-card-media"
                  image=""
                  title=""
                />,
                <WithStyles(CardContent)>
                  <WithStyles(Typography)
                    component="h2"
                    gutterBottom={true}
                    variant="h5"
                  >
                    North India Restaurant
                  </WithStyles(Typography)>
                  <WithStyles(Typography)
                    component="p"
                  >
                    Price: 
                     
                    <br />
                    Rating: 
                    4
                     
                    <br />
                    Review Counts: 
                    551
                  </WithStyles(Typography)>
                </WithStyles(CardContent)>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "className": "review-card-media",
                  "image": "",
                  "title": "",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(Typography)
                      component="h2"
                      gutterBottom={true}
                      variant="h5"
                    >
                      North India Restaurant
                    </WithStyles(Typography)>,
                    <WithStyles(Typography)
                      component="p"
                    >
                      Price: 
                       
                      <br />
                      Rating: 
                      4
                       
                      <br />
                      Review Counts: 
                      551
                    </WithStyles(Typography)>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "North India Restaurant",
                      "component": "h2",
                      "gutterBottom": true,
                      "variant": "h5",
                    },
                    "ref": null,
                    "rendered": "North India Restaurant",
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": Array [
                        "Price: ",
                        undefined,
                        " ",
                        <br />,
                        "Rating: ",
                        4,
                        " ",
                        <br />,
                        "Review Counts: ",
                        551,
                      ],
                      "component": "p",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Price: ",
                      undefined,
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Rating: ",
                      4,
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Review Counts: ",
                      551,
                    ],
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
